src/microbit/model/display.py

```diff
@@ -0,0 +1,208 @@
+import copy
+import time
+import threading
+
+from . import constants as CONSTANTS
+from .image import Image
+from .. import shim
+
+
+class Display:
+    # The implementation based off of https://github.com/bbcmicrobit/micropython/blob/master/docs/display.rst.
+    def __init__(self):
+        self.__image = Image()
+        self.__on = True
+
+    def scroll(self, value, delay=150, wait=True, loop=False, monospace=False):

```

Comment:
Did we check if the behavior is the same on the device and here when you call `show/scroll(_, wait=False)` followed shortly by a `show/scroll(_, wait=True)`?

Reply:
Changed it to work the same as microbit device